Remove analyzers which only make sense for the octopus server (#30)
OctopusDeploy/RoslynAnalyzers: readme.md

## Purpose

> This repository stores independent roslyn analyzers we use internally at Octopus Deploy which are shared across multiple projects.

Reviewer: ```suggestion
This repository stores roslyn analyzers we use at Octopus Deploy which are shared across multiple projects.
```

> For analyzers targeted at the Octopus Server codebase, please do not add them here. Rather, refer to OctopusDeploy/source/Octopus.Server.RoslynAnalyzers

Reviewer: ```suggestion
For analyzers that are specific to the Octopus Server codebase, please do not add them here. Rather, refer to OctopusDeploy/source/Octopus.Server.RoslynAnalyzers
```
